cmd/swarm/swarm-smoke/main.go

```diff
@@ -35,6 +42,17 @@ var (
 	from             int
 	to               int
 	verbosity        int
+	timeout          int
+)
+var (
+	feedUploadAndSyncCount      = gethmetrics.NewRegisteredCounter("swarm-smoke.feed-and-sync.count", nil)
```

Comment:
I think we should include the size of the file here. At least `time` will def be much different, depending on `size`.

I also still think this indirection is not necessary, you can increment the counter exactly at the place in the code inline, rather than define this variable.